helix-term/src/commands/typed.rs

```diff
@@ -551,6 +551,7 @@ fn write_all_impl(
     let auto_format = cx.editor.config().auto_format;
     let jobs = &mut cx.jobs;
     // save all documents
+    // Saves only visible buffers? How do we reload the not visible ones?
```

Comment:
This does save all documents, and on testing the non-visible ones are updated correctly too.

Reply:
Ah yeah that was from before we used transactions, will remove the comment in a bit